### ISSUE TITLE
Corrupted extract of Trinidad and Tobago

### ISSUE THREAD

**Issue by frodrigo** [osm-fr/infrastructure#280]
The extract only contains one marine cable
http://download.openstreetmap.fr/extracts/central-america/trinidad_and_tobago-latest.osm.pbf

**Comment by frodrigo** [comment]
The poly is good.

**Comment by jocelynj** [comment]
The issue was that the polygon for Central-America didn't contain Trinidad and Tobago. I've updated all global polygons from Geofabrik, on commit https://github.com/osm-fr/osm-extract-replication/commit/4ea8e8a8d80cb677fcc46c666373d8f708e98e1c.

I saw that are other differences, for example on South-America, so that islands belonging to Venezuela are in this polygon. So, I'm downloading the latest planet to regenerate all polygons.
